model_zoo/spleen_ct_segmentation_monai_bundle/README.md

```diff
@@ -0,0 +1,78 @@
+Introduction of BTCV dataset can be found in https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb.
+
+There are 13 organs labeld in BTCV. In this example, we focus on spleen segmentation only.
+
+Step 1: Download BTCV dataset RawData.zip following the instruction in https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb. Extract it as `./data/RawData`.
+
+Step 2: Download the the json file for data splits in https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb, and save it to `./data/dataset_0.json`.
+
+Step 3: The labels in BTCV dataset contains 13 organs. We split the labels and extract spleen label out. This is done by running `python ./split_spleen_labels.py`.
+
+Step 4: `cp -avr ./data/RawData/Training/img ./data/spleen/imagesTr`
+
+## Download example monai bundle from model-zoo
+```
+python -m monai.bundle download --name spleen_ct_segmentation --version "0.1.1" --bundle_dir "./"
+```
+
+## Training configuration
+The training was performed with at least 12GB-memory GPUs.
+During training, 19 out of the 24 scans are used for training, while the rest 5 is used for validation and model selection.
+
+Actual Model Input: 96 x 96 x 96
+
+## Modify ./spleen_ct_segmentation/configs/train.json from the downloaded example
+| Old json config | Updated json config |
+| --- | --- |
+| "bundle_root": "/workspace/data/tutorials/modules/bundle/spleen_segmentation", | "bundle_root": "./spleen_ct_segmentation", |
+| "dataset_dir": "/workspace/data/Task09_Spleen",| "data_file_base_dir": "./data/spleen", |
+| "images": "$list(sorted(glob.glob(@dataset_dir + '/imagesTr/*.nii.gz')))",| "data_list_file_path": "./data/dataset_0.json", |
+| "labels": "$list(sorted(glob.glob(@dataset_dir + '/labelsTr/*.nii.gz')))",| "train_datalist": "$monai.data.load_decathlon_datalist(@data_list_file_path, is_segmentation=True, data_list_key='training', base_dir=@data_file_base_dir)", |
+| (In "train#dataset",) "data": "$[{'image': i, 'label': l} for i, l in zip(@images[:-9], @labels[:-9])]",| "data": "$@train_datalist[: int(0.8 * len(@train_datalist))]", |
+| (In "validate#dataset",) "data": "$[{'image': i, 'label': l} for i, l in zip(@images[-9:], @labels[-9:])]",| "data": "$@train_datalist[int(0.8 * len(@train_datalist)):]", |
+| (In "train#trainer",) "max_epochs": 100,| "max_epochs": 600,|
+| (In "optimizer",) "lr": 0.0001,| "lr": 0.0002,|
+
+## Modify ./spleen_ct_segmentation/configs/evaluate.json
+| Old json config | Updated json config |
+| --- | --- |
+| (add)| "test_datalist": "$monai.data.load_decathlon_datalist(@data_list_file_path, is_segmentation=True, data_list_key='validation', base_dir=@data_file_base_dir)", |
+| (add)| "validate#dataset": {"_target_": "Dataset","data": "$@test_datalist","transform": "@validate#preprocessing"},|
```

Comment:
Maybe missing something?

Reply:
It means adding a new line to the json file. I have clarified it.

Reply:
OK, got it. Thanks for explanation!